src/main/java/javaClasses/homework_4/AndranikSargsyan/Calculations.java

```diff
@@ -0,0 +1,32 @@
+package javaClasses.homework_4.AndranikSargsyan;
+
+public class Calculations {
+
+    public int getMean(int... nums) {
+        int result = 0;
+        for (int n : nums)
+            result += n;
+        return result / nums.length;
+    }
+
+    public double getMean(double... nums) {
+        int result = 0;
+        for (double n : nums)
+            result += n;
```

Comment:
In this case you might lose some data. For example your n is 2.5 if you assign it to int result you'll store only 2. Please review that part, and on the other code parts below

Reply:
done